career/roles-in-tech/security.md

```diff
@@ -35,22 +35,22 @@ Apprentices will learn secure development basics, common pitfalls, and how to av
 
 _Participants will be able to learn:_
```

Comment:
These the Objectives should list what someone will know once they finish reading this outline.  Do you agree that this is what's listed?